lnrpc/lightning.proto

```diff
@@ -2733,6 +2733,10 @@ message PendingChannelsResponse {
 
         // The transaction id of the closing transaction
         string closing_txid = 4;
+
+        // The raw hex encoded bytes of the closing transaction. Can be used to
```

Comment:
nit: prefer to leave out the line `Can be used to...` and leave it up to the users to decide how to use it

Reply:
Done.